linera-base/src/identifiers.rs

```diff
@@ -10,6 +10,7 @@ use std::{
     str::FromStr,
 };
 
+use alloy_primitives::{Address, B256};
```

Comment:
Should this be guarded by a feature `evm` ?

Reply:
We can do by the feature `revm`. I introduced it in `linera-base`.
We have only one framework for EVM, so I did not introduce a feature `evm`.